keel-web/src/main/kotlin/com/netflix/spinnaker/keel/rest/AdminController.kt

```diff
@@ -49,4 +50,15 @@ class AdminController(
   fun triggerRecheck(@PathVariable("resourceId") resourceId: String) {
     adminService.triggerRecheck(resourceId)
   }
+
+  @PostMapping(
+    path = ["/application/{application}/environment/{environment}/reevaluate"]
+  )
```

Comment:
Suggestion:
```suggestion
  @DeleteMapping(
    path = ["/application/{application}/environment/{environment}/constraints"]
  )
```

Reply:
I think I'd like to keep this as a post, because while we are deleting something as the implementation we are not deleting the constraint. I think that `POST` conveys correctly that we're modifying something. What do you think about that reasoning @luispollo ?